masschallenge/django-accelerator
# [AC-9452] 1 Add program interest fields to Program model

**Changes made to [AC-9452](https://masschallenge.atlassian.net/browse/AC-9452):**
- Added fields in Program Model.

**How to test:**

- On this branch on accelerate, run `make migrate`
- Login or masquerade as a user with admin privileges.
- Checkout http://localhost:8181/admin.
- Ensure that you are able to see the fields in Program as indicated in the done of [AC-9452](https://masschallenge.atlassian.net/browse/AC-9452)

[Sibling PR](https://github.com/masschallenge/accelerate/pull/3255)
